apps/studio/src/features/editing-experience/components/LayoutSelection.tsx

```diff
@@ -0,0 +1,281 @@
+                      mb="1.25rem"
+                      position="relative"
+                      borderColor={
+                        selectedLayout.layoutTypename ===
+                        layoutEntry.layoutTypename
+                          ? 'base.divider.brand'
+                          : 'base.divider.medium'
+                      }
+                      bgColor={
+                        selectedLayout.layoutTypename ===
+                        layoutEntry.layoutTypename
+                          ? 'interaction.muted.main.active'
+                          : 'interaction.muted.main.hover'
+                      }
+                      sx={
+                        selectedLayout.layoutTypename !==
+                        layoutEntry.layoutTypename
+                          ? {
+                              transition:
+                                'border-color 300ms ease-out, opacity 300ms ease-out, background-color 300ms ease-out',
+                            }
+                          : {}
+                      }
+                      _groupHover={
+                        selectedLayout.layoutTypename !==
+                        layoutEntry.layoutTypename
+                          ? {
+                              borderColor: useToken(
+                                'colors',
+                                'interaction.main-subtle.hover',
+                              ),
+                              bgColor: useToken(
+                                'colors',
+                                'interaction.main-subtle.default',
+                              ),
+                            }
+                          : {}
+                      }
+                    >
+                      <Image
+                        src={layoutEntry.imageSource}
+                        borderRadius="4px 4px 0px 0px"
+                        boxShadow="md"
+                        alt={`Image of ${layoutEntry.layoutDisplayName} layout`}
+                      />
+
+                      {selectedLayout.layoutTypename !==
+                        layoutEntry.layoutTypename && (
+                        <>
+                          {/* for opacity, since we need the text to be clear */}
+                          <Box
+                            className="hover-blur"
+                            position="absolute"
+                            top="0"
+                            left="0"
+                            opacity="0.6"
+                            alignItems="center"
+                            w="100%"
+                            h="100%"
+                            borderRadius="6px" // due to this being smaller than the outer Box
+                            bgColor="interaction.main-subtle.default"
+                            sx={{
+                              transition: 'opacity 300ms ease-out',
+                              opacity: 0,
+                            }}
```

Comment:
might not need to mix `sx` and props, slightly affects readability

```suggestion
                            transition="opacity 300ms ease-out"
                            opacity={0}
```